packages/engine/src/xr/systems/XRSystem.ts

```diff
@@ -30,7 +31,15 @@ const startXRSession = async () => {
     dispatchLocal(EngineActions.xrSession() as any)
 
     // Current WebXRManager.getCamera() typedef is incorrect
-    ;(Engine.xrManager as any).getCamera().layers.enableAll()
+    // @ts-ignore
+    const cameras = Engine.xrManager.getCamera() as ArrayCamera
+    cameras.layers.enableAll()
+    cameras.cameras.forEach((camera) => {
```

Comment:
Not very sure but shouldn't we restore the camera layers to the previous state on `EngineEvents.EVENTS.XR_END`?